eng/restore/optimizationData.targets

```diff
@@ -0,0 +1,32 @@
+﻿<Project>
+
+  <ItemGroup>
+    <IBCPackage Include="$(WindowsCoreFxOptimizationDataPackageId)" Version="$(optimizationwindows_ntx64IBCCoreFxPackageVersion)" />
+    <IBCPackage Include="$(LinuxCoreFxOptimizationDataPackageId)" Version="$(optimizationwindows_ntx64IBCCoreFxPackageVersion)" />
+    <PackageReference Include="@(IBCPackage)" PrivateAssets="all" IsImplicitlyDefined="true" />
+  </ItemGroup>
+  
+  <!-- IBC data packages don't follow NuGet conventions.  -->
+  <Target Name="GetIBCData"
+          Inputs="%(IBCPackage.Identity)"
```

Comment:
I think you can remove the target level batching now

Reply:
I just tried it, without the inputs and outputs it only runs for one package, ie the linux one.

Reply:
Yeah, you need to change the content.  Here you go.
```
  <Target Name="GetIBCData"
          AfterTargets="ResolveReferences">
    <ItemGroup>
      <IBCPackage>
        <PackagePath>$(NuGetPackageRoot)%(IBCPackage.Identity)\%(IBCPackage.Version)</PackagePath>
      </IBCPackage>
      <_optimizationDataSourceFile Include="%(IBCPackage.PackagePath)\**\*.dll;%(IBCPackage.PackagePath)\**\*.ibc" Package="%(IBCPackage.Identity)" />
    </ItemGroup>
    <Copy SourceFiles="@(_optimizationDataSourceFile)"
          DestinationFiles="@(_optimizationDataSourceFile->'$(IbcOptimizationDataDir)%(Package)\%(RecursiveDir)%(Filename)%(Extension)')"
          SkipUnchangedFiles="true"
          UseHardlinksIfPossible="true" />
  </Target>
```